chameleon/settings.py

```diff
@@ -111,6 +112,7 @@
     ##
     # custom
     #
+    'chameleon',
```

Comment:
Is it OK for a Django site to load itself as an app? I thought that there was some difference in purpose between the "site" directory (./chameleon) and the app directories (e.g., ./chameleon_openid)

Reply:
OK, sounds like it's supported then :)